workspaces/javascript-leetcode-month/problems/2635-apply-transform-over-each-element-in-array/solutions/iterative-loops/using-classic-for-mutating-input.ts

```diff
@@ -0,0 +1,10 @@
+function map<TIn>(
+  arr: TIn[], 
+  fn: (element: TIn, index: number) => TIn
```

Comment:
It's probably missing the trailing comma.

Reply:
I was attempting to fix it, but I kept getting issues. I just ran `yarn format` instead.